src/datanode/src/instance/sql.rs

```diff
@@ -183,22 +183,25 @@ impl Instance {
 
                 Ok(Output::RecordBatches(RecordBatches::empty()))
             }
-            QueryStatement::Sql(Statement::Copy(copy_table)) => {
-                let (catalog_name, schema_name, table_name) =
-                    table_idents_to_full_name(copy_table.table_name(), query_ctx.clone())?;
-                let file_name = copy_table.file_name().to_string();
-
-                let req = CopyTableRequest {
-                    catalog_name,
-                    schema_name,
-                    table_name,
-                    file_name,
-                };
+            QueryStatement::Sql(Statement::Copy(copy_table)) => match copy_table {
+                sql::statements::copy::CopyTable::To(copy_table) => {
```

Comment:
Could you import sql::statements::copy::CopyTable to this file? This statement is so long.